### ISSUE TITLE
Test accuracy on cifar-10 0.2 sym noise

### ISSUE THREAD

**Issue by lxysl** [ragavsachdeva/ScanMix#2]
<img width="1694" alt="SCR-20230111-pma" src="https://user-images.githubusercontent.com/45818075/211840137-38d79fe6-d313-480d-a490-3ab0e41ea9d5.png">
Thanks for release the code and it inspired me! But I encountered a question.I use pretrained SCAN model to repeat the experiments on different noise rates, but I found when noise rate is 0.2, the test acc is lower than 0.5 and 0.8. What's wrong with that?

**Comment by ragavsachdeva** [comment]
Hi! To be honest, I am not sure. Can you consistently reproduce poor performance for 0.2 (in comparison to 0.5 and 0.8)? This project is something I worked on in 2020 (over 2years ago now) so I don't remember much of the implementation details I am afraid. The grey curve does seem to start of better than than others but then starts decreasing (after about 10epochs? Not clear from the plot). Something weird seems to be going on after the warmup.

Unfortunately, since last year I have graduated from my previous university and no longer have access the computer I used back then (with logs, conda env etc. - assuming they even exist anymore) so there is no easy way for me to test this. All I can confidently say is that in all my experiments I have never seen the above (0.2 noise rate's results were always better than 0.5 and 0.8, as in the paper).

My advice would be to plot/log more things (e.g. predicted noise rate, predicted distribution of noisy samples, learning rate, augmentation strategy etc.) and compare 0.2 with, say, 0.5.

Good luck and my apologies for not being able to offer more help.

**Comment by lxysl** [comment]
Thanks for your kind reply, I'll try more metrics to see what's wrong with it! Thank you for the outstanding work you've brought. Best wishes to you!

**Comment by lxysl** [comment]
I finally figured it out following [this issue](https://github.com/LiJunnan1992/DivideMix/issues/35)! It turns out the hyper-parameter lambda_u should be changed to 0 when noise ratio is 0.2.
